README.md

```diff
@@ -148,8 +153,11 @@ The following tools can be used to visualize genomic data or for constructing cu
 * __[IGV js](https://www.broadinstitute.org/igv/)__ - Java based browser. Fast, efficient, scalable visualization tool for genomics data and annotations. Handles a large [variety of formats](http://www.broadinstitute.org/igv/FileFormats). 
 * __[Island Plot](https://github.com/lairdm/islandplot)__ - D3 JavaScript based genome viewer. Constructs SVGs.
 * __[pileup.js](https://github.com/hammerlab/pileup.js)__ - JavaScript library that can be used to generate interactive and highly customizable web-based genome browsers.
+* __[JBrowse](http://jbrowse.org/)__ - JBrowse is a fast, embeddable genome browser built completely with JavaScript and HTML5, with optional run-once data formatting tools written in Perl.
 * __[scribl](https://github.com/chmille4/Scribl)__ - JavaScript library for drawing canvas-based gene diagrams. The [Homepage](http://chmille4.github.io/Scribl/) has examples.
 * __[DNAism](https://github.com/drio/dnaism)__ - Horizon chart d3-based js library for DNA data.
+* __[BioCircos.js](http://bioinfo.ibp.ac.cn/biocircos/)__ - BioCircos.js is an open source interactive Javascript library which provides an easy way to interactive display biological data on the web.
```

Comment:
There's a whole new section dedicated to Circos related tools. Also, change

- `Javascript` to `JavaScript`, 
- `open source interactive` to `open-source interactive`, and 
- `...an easy way to interactive...` to `...an easy way to create an interactive...`.